src/main/java/at/technikum/springrestbackend/security/user/UserPrincipal.java

```diff
@@ -0,0 +1,21 @@
+package at.technikum.springrestbackend.security.user;
+
+import lombok.Getter;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+public class UserPrincipal extends User {
+
+    private UUID id;
+    private String role;
```

Comment:
Shouldn't we put the password field here instead of the User class?

Reply:
Could you show me how you would change it?